lib/models/product_list.dart

```diff
@@ -6,9 +6,33 @@ class ProductList with ChangeNotifier {
   List<Product> _items = dummyProducts;
 
   List<Product> get items => [..._items];
+  List<Product> get favoriteItems =>
+      _items.where((prod) => prod.isFavorite).toList();
 
   void addProduct(Product product) {
     _items.add(product);
     notifyListeners();
   }
-}
\ No newline at end of file
+}
+
+/// this commented code bellow is the previous filter
```

Comment:
Remove commented code

Reply:
comment is a way to see how the other version of the filter worked therefore it won't be removed

Reply:
No commented code should be left, if you need to see previous version use git to do it